packages/lpa-questionnaire-web-app/src/middleware/already-submitted.js

```diff
@@ -0,0 +1,26 @@
+const { validate: validateUuid } = require('uuid');
+const { VIEW } = require('../lib/views');
+
+module.exports = (req, res, next) => {
+  const { id: appealId = '' } = req.params;
+
+  if (!appealId || !validateUuid(appealId)) {
+    res.status(404).send();
+    return;
+  }
+
+  const questionnaire = req.session?.appealReply;
+
+  if (!questionnaire) {
+    req.log.error('Error retrieving questionnaire');
+    res.status(404).send();
+    return;
+  }
+
+  if (questionnaire.state.toUpperCase() === 'SUBMITTED') {
```

Comment:
Do we need to do a comparison with the upper case version of submitted?

Reply:
@demzzydev  Removed the case check